addons/html_editor/controllers/main.py

```diff
@@ -570,15 +570,20 @@ def link_preview_metadata_internal(self, preview_url):
 
             record_id = int(words.pop())
             action_name = words.pop()
-            action = Actions.sudo().search([('path', '=', action_name)])
-            if not action:
-                return {'error_msg': _("Action %s not found, link preview is not available, please check your url is correct", action_name)}
-            action_type = action.type
-            if action_type != 'ir.actions.act_window':
-                return {'other_error_msg': _("Action %s is not a window action, link preview is not available", action_name)}
-            action = request.env[action_type].browse(action.id)
-
-            model = request.env[action.res_model].with_context(context)
+            if action_name in request.env:
```

Comment:
I'm not sure if we want to add this here to be able to access the model directly when the model name is given because it will may access some other models like `ir.http`, `bus.bus`.... We need to check the spec here if we want to support this or not . 
cc @Jinjiu96 , @dmo-odoo

Reply:
Hello @Mtaylorr Thanks for the catch, indeed we should restrict that.
Thanks for the suggestion @Jinjiu96 , i was not sure how to handle it in a safe way. i updated the commit please let me know if i missed anything.